core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy

```diff
@@ -367,6 +367,46 @@ class TomcatConfigurator {
     return this
   }
 
+  /**
+   * Removes the specified listener to server.xml
+   * @param listenerFQCN the fully qualified class name of the listener
+   * @return this
+   */
+  TomcatConfigurator removeListener(String listenerFQCN) {
+    String serverXml = 'server.xml'
```

Comment:
Is there any reason to not use `getServerXml()` ?

Reply:
No specific reason.Used the same code as in the "addListener" in order to match in the eye. After all i agree that getServerXml() is better and less code

Reply:
Thank you